Ghidra/Features/GraphServices/build.gradle

```diff
@@ -11,12 +11,13 @@ eclipse.project.name = 'Features Graph Services'
 dependencies {
 	compile project(":Base")
 
-	compile "com.github.tomnelson:jungrapht-visualization:1.0-RC7"
+	compile "com.github.tomnelson:jungrapht-visualization:1.0-RC8"
 	compile "org.jgrapht:jgrapht-core:1.4.0"
 	
 	// not using jgrapht-io code that depends on antlr, so exclude antlr
 	compile ("org.jgrapht:jgrapht-io:1.4.0") { exclude group: "org.antlr", module: "antlr4-runtime" }
 	runtime "org.slf4j:slf4j-api:1.7.25"
+	runtime "org.slf4j:slf4j-nop:1.7.25"
```

Comment:
So the warning about logging provider is not sent to console. A log4j impl could be used instead, to log jungrapht according the Ghidra log levels.